deployment-examples/kubernetes/00_infra.sh

```diff
@@ -77,50 +80,41 @@ kubectl wait --for condition=Established crd/referencegrants.gateway.networking.
 # Start cilium.
 
 helm repo add cilium https://helm.cilium.io
-
+helm repo update cilium
 helm upgrade \
     --install cilium cilium/cilium \
-    --version 1.15.0-pre.3 \
+    --version 1.14.5 \
     --namespace kube-system \
     --set k8sServiceHost=kind-control-plane \
     --set k8sServicePort=6443 \
     --set kubeProxyReplacement=strict \
     --set gatewayAPI.enabled=true \
+    --set l2announcements.enabled=true \
     --wait
 
-# Set up MetalLB. Kind's nodes are containers running on the local docker
-# network. We reuse that network for LB-IPAM so that LoadBalancers are available
-# via "real" local IPs.
+# Kind's nodes are containers running on the local docker network. We reuse that
+# network for LB-IPAM so that LoadBalancers are available via "real" local IPs.
 
 KIND_NET_CIDR=$(docker network inspect kind -f '{{(index .IPAM.Config 0).Subnet}}')
-METALLB_IP_START=$(echo ${KIND_NET_CIDR} | sed "s@0.0/16@255.200@")
-METALLB_IP_END=$(echo ${KIND_NET_CIDR} | sed "s@0.0/16@255.250@")
-METALLB_IP_RANGE="${METALLB_IP_START}-${METALLB_IP_END}"
-
-helm install --namespace metallb-system --create-namespace \
-  --repo https://metallb.github.io/metallb metallb metallb \
-  --version 0.13.12 \
-  --wait
+CILIUM_IP_CIDR=$(echo ${KIND_NET_CIDR} | sed "s@0.0/16@255.0/28@")
 
 cat <<EOF | kubectl apply -f -
 ---
-apiVersion: metallb.io/v1beta1
-kind: L2Advertisement
+apiVersion: cilium.io/v2alpha1
+kind: CiliumL2AnnouncementPolicy
```

Comment:
This is less stable not more stable, right? I think it's fine, but we need to ensure that neither our CI our future releases are impeded by an Alpha announcement policy.